examples/get_started/04_table_inputs.py

```diff
@@ -0,0 +1,148 @@
+"""
+4. PyGMT I/O: Table inputs
+==========================
+
+Generally, PyGMT accepts two different types of data inputs: tables and grids.
+
+- A table is a 2-D array of data, with *M* rows and *N* columns. Each column
+  represents a different variable (e.g., *x*, *y* and *z*) and each row
+  represents a different record.
+- A grid is a 2-D array of data that is regularly spaced in the x and y
+  directions.
+
+In this tutorial, we'll focus on working with table inputs, and cover grids in
+the following tutorials.
+
+PyGMT supports a variety of table input types that allow you to work with data
+in a format that suits your needs. In this tutorial, we'll explore the
+different table input types available in PyGMT and provide examples for each.
+By understanding the different table input types, you can choose the one that
+best fits your data and analysis needs, and work more efficiently with PyGMT.
+"""
+
+# %%
+# ASCII table file
+# ----------------
+#
+# Most PyGMT functions/methods that accept table input data have a ``data``
+# parameter. The easiest way to provide table input data to PyGMT is by
+# specifying the file name of an ASCII table (e.g., ``data="input_data.dat"``).
+# This is useful when your data is stored in a separate text file.
+
+import numpy as np
+import pygmt
```

Comment:
Hope this is the correct order...

```suggestion
from pathlib import Path

import geopandas as gpd
import numpy as np
import pandas as pd
import pygmt
```